lib/susedistribution.pm

```diff
@@ -264,7 +264,7 @@ sub init_consoles {
         $self->add_console('installation',   'tty-console', {tty => check_var('VIDEOMODE', 'text') ? 1 : 7});
         $self->add_console('install-shell2', 'tty-console', {tty => 9});
         # On SLE15 X is running on tty2 see bsc#1054782
-        $self->add_console('root-console', 'tty-console', {tty => sle_version_at_least('15') ? 6 : 2});
+        $self->add_console('root-console', 'tty-console', {tty => sle_version_at_least('15') && is_sle ? 6 : 2});
```

Comment:
IMHO sle_version_at_least should internally only check for `is_sle` but we would need to check where it makes problems